model/variable.h

```diff
@@ -129,7 +130,21 @@ namespace minsky
       if (auto v=vValue()) return v->hypercube().dims();
       else return {};
     }
-      
+
+    std::vector<std::string> dimLabels() const {
+      if (auto v=vValue()) {
+        std::vector<std::string> tmpLabels;			  
+        for (auto& i: v->hypercube().xvectors) tmpLabels.push_back(static_cast<std::string>(i.name));
+        return tmpLabels;   
```

Comment:
Maybe useful enough operation to make a member of Hypercube - perhaps call it Hypercube::dimNames()?

Reply:
BTW - the static_cast is not required here.

Reply:
> Maybe useful enough operation to make a member of Hypercube - perhaps call it Hypercube::dimNames()?

Very good suggestion, thanks. Going to try to move all the new methods I added to VariableBase to Hypercube.

Reply:
> BTW - the static_cast is not required here.

True, getting rid of it :)